frontend/src/apis/index.js

```diff
@@ -0,0 +1,39 @@
+// Is an abstraction to communicate with backend
+
+class API {
+    BASE_URL = '/api';
+    myFetch = (url, config) =>
+        window.fetch(this.BASE_URL + url, {
+            headers: { 'content-type': 'application/json' },
+            ...config,
+        });
+
+    async login({ pin, phoneNumber }) {
+        try {
```

Comment:
The errors don't need to be catched within the methods just to be rethrown without modification. Also the functions don't need to be `async` functions.